### ISSUE TITLE
[Feature Request] Add metadata to chapters

### ISSUE THREAD

**Issue by Vodes** [dazedcat19/FMD2#142]
It would be very helpful if the files had the metadata for things like author and summary. This should be fairly easy to implement atleast for the epub standard. See [this](https://wiki.mobileread.com/wiki/Metadata) for example

**Comment by Inpacchi** [comment]
I just developed an application specifically for this and meant to work as an extension to FMD. Unfortunately I don’t know Object Pascal otherwise I would’ve integrated it directly into FMD. 

https://github.com/inpacchi//manga-tagger

**Comment by Vodes** [comment]
> 
> 
> I just developed an application specifically for this and meant to work as an extension to FMD. Unfortunately I don’t know Object Pascal otherwise I would’ve integrated it directly into FMD.
> 
> https://github.com/inpacchi//manga-tagger

Yes i've seen that already and once Linux support is there, i will use it ^^
